packages/backend/src/services/order.ts

```diff
@@ -69,4 +69,22 @@ export class OrderService {
   public async getById(id: string): Promise<Order & OrderById> {
     return this._orderRepository.getById(id);
   }
+
+  public isPossibleToCreateOrder(product: Product): boolean {
```

Comment:
why public?

Reply:
I planned to use it outside but then do it in other way. I will change to private